.changeset/seven-owls-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/select-utils': patch
+---
+
+Added min-width property to `SelectInput` component
```

Comment:
Sorry to be picky but maybe we can use another working as this might give the impression that we're adding a new property to the component when we're just adding a style rule.

Suggestion:

Updated `SelectInput` based components so the dropdown layer has a minimum width ([`constraint 3`](https://uikit.commercetools.com/?path=/story/components-constraints--horizontal))

Reply:
@CarlosCortizasCT good point thanks! Updated here - [270db7d](https://github.com/commercetools/ui-kit/pull/2654/commits/270db7d7def8f7d5982c14fac2b4aa76861e5c52).